webpack.config.js

```diff
@@ -16,6 +16,18 @@ module.exports = {
       import: './src/displayCommentPopup.js',
       dependOn: 'shared',
     },
+    getComments: {
+      import: './src/getComments.js',
+      dependOn: 'shared',
+    },
+    getCardList: {
+      import: './src/getCardsApiData.js',
+      dependOn: 'shared',
+    },
+    displayList: {
+      import: './src/displayList.js',
+      dependOn: 'shared',
+    },
```

Comment:
I'm curious, this code is new to me, it looks interesting.
Would you mind telling me about the benefits that this brings to the project?

Reply:
It is to prevent duplication of modules between the chunks. I followed [this](https://webpack.js.org/guides/code-splitting/#prevent-duplication) page's instructions.